Realistic_Electric_Trains_0.3.0/logic/events/on_tick.lua

```diff
@@ -9,16 +9,30 @@ do
 	function on_tick(event)
 
 		if not loco_fuel_data then
+			local loco_1_prototype = game.entity_prototypes["ret-electric-locomotive"]
+			local loco_1_efficiency_modifier = 1.05
+			if loco_1_prototype.burner_prototype.effectivity < 1 then
+				loco_1_efficiency_modifier = loco_1_efficiency_modifier * (2 - loco_1_prototype.burner_prototype.effectivity)
+			end
+			
+			local loco_2_prototype = game.entity_prototypes["ret-electric-locomotive-mk2"]
+			local loco_2_efficiency_modifier = 1.05
+			if loco_2_prototype.burner_prototype.effectivity < 1 then
+				loco_2_efficiency_modifier = loco_2_efficiency_modifier * (2 - loco_2_prototype.burner_prototype.effectivity)
```

Comment:
Same here obviously :D
```suggestion
				loco_2_efficiency_modifier = loco_2_efficiency_modifier / loco_2_prototype.burner_prototype.effectivity
```